godtossing/Pos/Ssc/GodTossing/Toss/Trans.hs

```diff
@@ -76,7 +75,7 @@ instance MonadToss m =>
     putShares id sh =
         ether $ tmShares . at id .= Just sh
     putCertificate cert =
-        ether $ tmCertificates %= HM.insert (getCertId cert) cert
+        ether $ tmCertificates %= fst . insertVss cert
```

Comment:
It seems to be ok to discard the second part of result here, because it should be always empty (ensured by `checkCertificatesPayload`). It would be good to write a brief comment explaining it.